XDMA/linux-kernel/xdma/xdma_mod.c

```diff
@@ -194,7 +232,7 @@ static int probe_one(struct pci_dev *pdev, const struct pci_device_id *id)
 		if (rv)
 			goto err_out;
 	}
-
+        
```

Comment:
Lint this

Reply:
Please check the file diff before you submit the PR
Also on commit (git commit **-v**)